2017/CG-11.md

```diff
@@ -77,12 +77,14 @@ A list of near by hotels:
 * Thursday - November 2nd
     1. Find volunteers for note taking
     1. Proposals and discussions
-       1. TBD
+       1. Update on proposals from July: Multi-value (Andreas Rossberg)
+            1. [Proposal Repo](https://github.com/WebAssembly/multi-value)
+            1. POLL: Should this enter the [Implementation Phase](https://github.com/WebAssembly/meetings/blob/master/process/phases.md#3-implementation-phase-community--working-group)?
     1. Closure
 
 ### Schedule constraints
 
-*None*
+Andreas Rossberg: only available on Thursday (via VC)
```

Comment:
Would you want someone from your team to present in-person? It might be easier than over VC?

Reply:
Will think about it. May be okay though.